src/app/src/components/EmbeddedMapFieldsConfig.jsx

```diff
@@ -63,6 +66,17 @@ const styles = {
     textInput: {
         width: '250px',
     },
+    searchable: {
+        marginLeft: '15px',
+    },
+    icon: {
+        color: 'rgba(0, 0, 0, 0.38)',
+        verticalAlign: 'super',
+        fontSize: '12px',
+    },
```

Comment:
With the new icon, you can actually remove `verticalAlign` and `fontSize` entirely from `.icon` and it'll appear quite nice.

```suggestion
    icon: {
        color: 'rgba(0, 0, 0, 0.38)',
    },
```

Reply:
That does look better!